pkg/kubelet/pod/mirror_client_test.go

```diff
@@ -54,3 +57,76 @@ func TestParsePodFullName(t *testing.T) {
 		}
 	}
 }
+
+func TestIsStaticPod(t *testing.T) {
+	testcases := []struct {
+		pod    *v1.Pod
+		expect bool
+	}{
+		{
+			pod: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						kubetypes.ConfigSourceAnnotationKey: kubetypes.ApiserverSource,
+					},
+				},
+			},
+			expect: false,
+		},
+		{
+			pod: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						kubetypes.ConfigSourceAnnotationKey: kubetypes.ApiserverSource + "0",
+					},
+				},
+			},
+			expect: false,
+		},
+		{
+			pod: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						kubetypes.ConfigSourceAnnotationKey: kubetypes.ApiserverSource + "99",
+					},
+				},
+			},
+			expect: false,
+		},
+		{
+			pod: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{},
+				},
+			},
+			expect: false,
+		},
+		{
+			pod: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						"abc": "def",
+					},
+				},
+			},
+			expect: false,
+		},
+		{
+			pod: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						kubetypes.ConfigSourceAnnotationKey: "abc",
+					},
+				},
+			},
+			expect: true,
+		},
+	}
+
+	for i, test := range testcases {
+		actual := IsStaticPod(test.pod)
+		if test.expect != actual {
+			t.Errorf("case %d faile: expected %v, got %v", i, test.expect, actual)
```

Comment:
typo "faile"

Reply:
sin of copy paste. fixed here an 3 other places.